span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/util/AttributeValueCreator.java

```diff
@@ -28,4 +29,22 @@
 
     return valueBuilder.build();
   }
+
+  public static JaegerSpanInternalModel.KeyValue convertAttributeToKeyValue(
```

Comment:
Should we move this to some other file?

Reply:
The first method in the class is for converting `KeyValue` to `AttributeValue`, the newly added method is doing the reverse.

Reply:
However the class can be renamed to say `JaegerHTTagsConverter` thus signifying that this class convert Tag to/from Jaeger to HT format

Reply:
Yeah, the name was confusing, and in the diff, I didn't see the above method. But, now it makes sense. It is a helper class that is converting to/from each other.